operatorconfig/driverconfig/powerstore/v2.5.0/controller.yaml

```diff
@@ -240,7 +240,7 @@ spec:
             - name: X_CSI_DRIVER_NAME
               value: "csi-powerstore.dellemc.com"
             - name: X_CSI_POWERSTORE_EXTERNAL_ACCESS
-              value: "None"
+              value: <X_CSI_POWERSTORE_EXTERNAL_ACCESS>
```

Comment:
@karthikk92, should this change be done for 2.4 yaml files as well

Reply:
take care